<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/logo.png</normaloff>:/res/logo.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Recent_Files"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="action_Telegram"/>
    <addaction name="action_Reddit"/>
    <addaction name="separator"/>
    <addaction name="action_Report_an_Issue"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Listing"/>
    <addaction name="action_Segments"/>
    <addaction name="action_Functions"/>
    <addaction name="action_Exports"/>
    <addaction name="action_Imports"/>
    <addaction name="action_Strings"/>
    <addaction name="separator"/>
    <addaction name="action_Reset_Layout"/>
   </widget>
   <widget class="QMenu" name="menu_REDasm">
    <property name="title">
     <string>&amp;REDasm</string>
    </property>
    <addaction name="action_Database"/>
    <addaction name="separator"/>
    <addaction name="action_Developer_Tools"/>
    <addaction name="action_Settings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_REDasm"/>
   <addaction name="menu_Window"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Telegram">
   <property name="text">
    <string>&amp;Telegram</string>
   </property>
  </action>
  <action name="action_Reddit">
   <property name="text">
    <string>&amp;Reddit</string>
   </property>
  </action>
  <action name="action_Report_an_Issue">
   <property name="text">
    <string>&amp;Report an Issue</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Recent_Files">
   <property name="text">
    <string>&amp;Recent Files</string>
   </property>
  </action>
  <action name="action_Reset_Layout">
   <property name="text">
    <string>&amp;Reset Layout</string>
   </property>
  </action>
  <action name="action_Segments">
   <property name="text">
    <string>&amp;Segments</string>
   </property>
  </action>
  <action name="action_Functions">
   <property name="text">
    <string>&amp;Functions</string>
   </property>
  </action>
  <action name="action_Listing">
   <property name="text">
    <string>&amp;Listing</string>
   </property>
  </action>
  <action name="action_Database">
   <property name="text">
    <string>&amp;Database</string>
   </property>
  </action>
  <action name="action_Exports">
   <property name="text">
    <string>&amp;Exports</string>
   </property>
  </action>
  <action name="action_Imports">
   <property name="text">
    <string>&amp;Imports</string>
   </property>
  </action>
  <action name="action_Strings">
   <property name="text">
    <string>&amp;Strings</string>
   </property>
  </action>
  <action name="action_Developer_Tools">
   <property name="text">
    <string>&amp;Developer Tools</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
